chainercv/utils/image/tile_images.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import division
+
+import math
+import numpy as np
+
+
+def tile_images(imgs, n_col, pad=2, fill=0):
+    """Make a tile of images
+
+    Args:
+        imgs (numpy.ndarray): A batch of images whose shape is BCHW.
+        n_col (int): Number of columns in a tile.
+        pad (int): Amount of pad. Default is 2.
+        fill (float, tuple or ~numpy.ndarray): The value of padded pixels.
+            If it is :class:`numpy.ndarray`,
+            its shape should be :math:`(C, 1, 1)`,
+            where :math:`C` is the number of channels of :obj:`img`.
+
+    Returns:
+        ~numpy.ndarray:
+        An image array in CHW format.
+
+    """
+    B, C, H, W = imgs.shape
+    n_col = min(n_col, B)
+    n_row = int(math.ceil(B / n_col))
+
+    shape = (C,
+             (H + pad) * n_row,
+             (W + pad) * n_col)
+    tile = np.empty(shape, dtype=imgs.dtype)
+    tile[:] = np.array(fill).reshape((-1, 1, 1))
+
+    k = 0
+    for y in range(n_row):
+        for x in range(n_col):
+            if k >= B:
+                break
+            start_y = y * (H + pad) + pad // 2 + 1
```

Comment:
Why do you add `1`? In the case that `pad == 0`, the top-left corner of the first image will be `(1, 1)` although it should be `(0, 0)`.